### DISCUSSION TITLE
Use of Flickity Fade in the card view

### DISCUSSION THREAD

**Discussion by eric-duncan-pdx** [microsoft-search/pnp-modern-search#2277] [Ideas]
From the Flickity documentation, it appears relatively straightforward to using Flickity Fade in the card view rather than the traditional "carousel" view.  I have yet to get it to work as expected, however.  Has anyone been successful in getting that to work or is there a package that needs to be incorporate during the build step in order to utilize that?

https://flickity.metafizzy.co/options.html#fade

**Comment by FranckyC** [comment]
Hi @eric-duncan-pdx, Flickity options are not all exposed in Web Part settings. You need to update the code directly to get this behavior https://github.com/microsoft-search/pnp-modern-search/blob/main/search-parts/src/components/SliderComponent.tsx . You could also expose this parameter in the slider layout. Feel free to PR; )
